.rubocop.yml

```diff
@@ -8,13 +8,41 @@ AllCops:
     - "Rakefile"
     - "yard/**/*"
     - "website/**/*"
+Metrics/AbcSize:
+  Exclude:
+    - "lib/signet.rb"
+    - "lib/signet/oauth_1.rb"
+    - "lib/signet/oauth_1/client.rb"
+    - "lib/signet/oauth_1/credential.rb"
+    - "lib/signet/oauth_1/server.rb"
+    - "lib/signet/oauth_2/client.rb"
```

Comment:
With so many of the significant files excluded, what do you think about just disabling these cops instead, or maybe setting them to the upper limits that allow all files to pass?

Reply:
Yeah good point. I started by trying just to exclude a minimal set of files, but now that's pretty much all of them. I'll look at upping the limits for now.

Reply:
That's what I prefer. As a rule, I'd rather up the limits for all files than set special excludes (or even worse, inline disables). It's more democratic: If one file deserves extra AbcSize, we all deserve extra AbcSize!